lib/src/locale/locale.dart

```diff
@@ -135,7 +141,8 @@ const _locales = <String, DurationLocale>{
   'it': italianLocale,
   'de': germanLocale,
   'nl': dutchLocale,
-  'zh': chineseLocale,
+  'zh': chineseSimplifiedLocale,
```

Comment:
Should this be 'zh' or 'zh-Hans'

Reply:
I kept zh because this will be a breaking change unless you don't mind pushing a major version upgrade